src/api.js

```diff
@@ -0,0 +1,82 @@
+const axios = require('axios');
+const rateLimit = require('axios-rate-limit');
+require('dotenv').config({path: '../.env'})
+const { ETHERSCAN_API_KEY, ETHERSCAN_API_URL, FAUCET_ADDRESS, GOERLI_API_URL } = process.env;
+
+const getBlockNumber = async function(time) {
+    const request = await rateLimit(axios.create(), {maxRequests: 5, perMillisecondss: 1000, maxRPS: 1})
+    const url = `${GOERLI_API_URL}?module=block&action=getblocknobytime&timestamp=${time}&closest=before&apikey=${ETHERSCAN_API_KEY}`
+    return (await (request.get(url))).data.result;
+   
+}
+const getTransactions = async function (address, fromBlock) {
+    const request = await rateLimit(axios.create(), {maxRequests: 5, perMillisecondss: 1000, maxRPS: 1})
+    const url = `${GOERLI_API_URL}?module=account&action=txlist&address=${address}&startblock=${fromBlock}&endblock=latest&sort=desc&apikey=${ETHERSCAN_API_KEY}`
+    const res = await request.get(url)
+    return res.data.result;
+}
+
+const getBalance = async function (address){
+    const request = await rateLimit(axios.create(), {maxRequests: 5, perMillisecondss: 1000, maxRPS: 1});
+    const url = `${GOERLI_API_URL}?module=account&action=balance&address=${address}&tag=latest&apikey=${ETHERSCAN_API_KEY}`;
+    return (await request.get(url)).data.result;
+}
+
+module.exports = {
+    checkDeposit: async function(address) {
+        var paidFlag = false;
```

Comment:
If it is never re-defined it should be definitely a constant, not variable.

Reply:
Variable removed!